coderkevin/fresh-data-wp-plugin-example
# Add API update/create for tag

This adds the ability to update a tag via the UI and API.

Known issue: It doesn't update the original data from the page fetch
yet. Changes to fresh-data need to happen for this.

To Test:
0. Clone `fresh-data` and do `npm install` and `npm link`.
1. Clone this repo under wp-content/plugins in a WP dev install.
2. `npm install`
3. `npm link fresh-data`
4. `npm start`
5. `Enable the plugin`
6. `Visit the "Fresh Data 🥕" admin menu item.`
7. Verify you see the "Tags" heading and the list of tags from you site appear underneath it within a couple of seconds.
8. Click to edit a tag.
9. Edit the name or description of the tag.
10. Click "Save"
11. Tag will appear as it did before, but wait up to 5 minutes and it will update (see "Known issue" above).
